pyhub/mcptools/microsoft/excel/tests/test_com_stability.py

```
@@ -0,0 +1,334 @@
+            # Add a workbook
+            wb = app.books.add()
+            sheet = wb.sheets.active
+
+            # Write some data
+            sheet.range("A1").value = "Test"
+
+            # Force quit Excel (simulating crash)
+            app._app.Quit()
+
+            # Try to use it again - should raise error
+            with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)